solr/core/src/test/org/apache/solr/search/TestXmlQParser.java

```diff
@@ -18,20 +18,23 @@
 
 import java.lang.invoke.MethodHandles;
 
+import org.apache.lucene.analysis.MockAnalyzer;
+import org.apache.lucene.analysis.MockTokenFilter;
+import org.apache.lucene.analysis.MockTokenizer;
 import org.apache.lucene.queryparser.xml.CoreParser;
 
-import org.apache.lucene.queryparser.xml.TestCoreParser;
-
+import org.apache.solr.SolrTestCase;
 import org.apache.solr.util.StartupLoggingUtils;
 import org.apache.solr.util.TestHarness;
 
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-
-public class TestXmlQParser extends TestCoreParser {
+@Ignore("Was relying on Lucene test sources. Should copy?")
```

Comment:
SolrCoreParser extends (Lucene) CoreParser. (Solr) XmlQParser[Plugin] uses SolrCoreParser. The extending and use are (currently only) to support user-customisation via configuration of the classes, hence the `testSomeOtherQuery()` below is commented out and the `TestXmlQParser` test here doesn't test anything beyond what it inherits from the `TestCoreParser` test.

I think it's okay to just delete the `TestXmlQParser` test.

code links:

* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/lucene/queryparser/src/java/org/apache/lucene/queryparser/xml/CoreParser.java
* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/lucene/queryparser/src/test/org/apache/lucene/queryparser/xml/TestCoreParser.java

* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/solr/core/src/java/org/apache/solr/search/SolrCoreParser.java
* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/solr/core/src/test/org/apache/solr/search/TestSolrCoreParser.java

* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/solr/core/src/java/org/apache/solr/search/XmlQParserPlugin.java
* https://github.com/apache/lucene-solr/blob/releases/lucene-solr%2F8.8.1/solr/core/src/test/org/apache/solr/search/TestXmlQParserPlugin.java

Reply:
Thanks Christine. I'll just remove it.

Reply:
Just opened #2465 for the removal before seeing your comment here (but the JIRA ticket).